.github/workflows/lint-test.yaml

```diff
@@ -23,10 +22,8 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Set up Helm
+      - name: Setup Helm
         uses: azure/setup-helm@v3
-        with:
-          version: v3.12.0
```

Comment:
According to https://github.com/Azure/setup-helm?tab=readme-ov-file#example, the default version is the newest, which we want to use.

Reply:
Good catch, the action claims to default to the latest version tho.